### DISCUSSION TITLE
Vector Level geometric tool snap too high

### DISCUSSION THREAD

**Discussion by optical-core** [opentoonz/opentoonz#6371] [Q&A]
### Is your feature request related to a problem? Please describe

![aa](https://user-images.githubusercontent.com/39340555/229259661-6fc6986d-bbcf-4101-a5b9-07aa5553ed9a.png)
Vector Level geometric tool snap "Low" mode too high. 

### Describe the solution you'd like

Snap adds "Very Low" mode.

**Comment by RodneyBaker** [comment]
It might be that this feature (as well as many others) would be best served not by adding new options but rather by allowing the user to set the values for each of the currently available options.

I find this request from you @optical-core to be particularly interesting in that I almost never use the low setting for Snap (in any of it's various locations) because it is too weak to effectively snap to a target.  That you find the low setting to need to be even lower is therefore very interesing to me as I wouldn't have ever considered having that value be even lower.

Interesting.  :)

Added:  I do note that you are using Geometry Tool here and that might be the real point of differentiation.

**Comment by optical-core** [comment]
I found that the snap range is related to dpi,
When the scene is 50dpi, the snap range is low;
When the scene is 200dpi, the snap range is too high;
That is to say, the snap range is in units of dpi, not pixels,
In the 200dpi scene, [snap low] still has a significant impact on drawing fine vector level.

**Comment by ghost** [comment]
Well, I'm not the only one who has this feeling...I made a similar request here #3271. The problem often occurs for the drawing of the eyes in my case because they are often surrounded by strokes of another part of the face like the hair or the nose...And so I snap very often on the wrong line. 

Another problem that can be linked to this one is the closing of a shape with the geometry tools (polyline, multi arc...) where there are snap of closing shapes...automatic. This forces you to close a shape without wanting to because the snap is too strong even in low. 

And the experiment is also present on the normal brush. The "too strong" low snap is quite noticeable when you bring 2 strokes together without joining them, the stroke is as if magnetized while it still has a long way to go before touching the other stroke.

A very low snap or a slider to change the snap level would be a plus. I never encounter this problem in cacani for example. The snap fits perfectly. And really snap at the last moment at the contact point
